### ISSUE TITLE
Check that all files generated by current setup are also generated here.

### ISSUE THREAD

**Issue by JJ** [antoniogamiz/Perl6-Documentable#106]
The files generated now are in https://github.com/perl6docs/perl6docs.github.io/

Write an auxiliary script that checks that.

**Comment by antoniogamiz** [comment]
With this script you can get all links:

~~~perl6
my @html;

# standard files
for <language programs routine syntax type> {
    @html.append: dir($_)
}
@html.grep(/\.html/);

# indexes
my @indexes = dir(".").grep(/^routine || ^type || ^syntax || ^language/);
spurt "links-from-io.txt", @html.join("\n")
~~~

**Comment by antoniogamiz** [comment]
I have obtained the following output: https://gist.github.com/antoniogamiz/8d3b2d0e7202d62d991ffb30a5216a68

New links does not matter. Let's see missing ones:

- This kind of files: `language/001-GroupA.html` are generated by `managed-page-order.p6`, which is not used by Perl6::Documentable.
- `syntax/ foo.html` are no longer generated because the problem with spaces is already solved.
- `type/independent-routines` has a problem with metadata. See https://github.com/perl6/doc/issues/2973.
- `routine/is DEPRECATED (Attribute).html` => (Attribute) part has been removed to create better links.
- `routine/term:<>.html` => Discussed in https://github.com/perl6/doc/issues/2975
